src/azul/project/hca/transformers.py

```diff
@@ -877,3 +907,9 @@ def _get_accumulator(self, field) -> Optional[Accumulator]:
             return FrequencySetAccumulator(max_size=100)
         else:
             return SetAccumulator()
+
+
+class SequencingProcessAggregator(SimpleAggregator):
+
+    def _get_accumulator(self, field) -> Optional[Accumulator]:
+        return SetAccumulator(max_size=10)
```

Comment:
Is 10 really the appropriate `max_size` value here? Every other usage of `max_size` in this file it set to 100, so this looks like a missing zero. If 10 really is the correct value, consider adding a comment explaining why.

Reply:
I did specify 10 in the ticket. The reason for it being so low is that we don't expect more than 10 processes (inner entitiy) being linked to each file (outer entity). For other outer entities there may hundreds or thousands of processes but we don't care about them for those types, only the `files` type is important because it is used to generate the manifests. It might make sense to capture this reasoning in a comment.